runtime/CMakeLists.txt

```diff
@@ -117,24 +115,21 @@ target_include_directories(${PROJECT_NAME}
   ${CMAKE_CURRENT_BINARY_DIR}
   ${CMAKE_CURRENT_SOURCE_DIR}/../third_party
   ${SPDLOG_INCLUDE}
-  ${FRIDA_UPROBE_ATTACH_IMPL_INCLUDE}
-  ${SYSCALL_TRACE_ATTACH_IMPL_INCLUDE}
 )
 
 target_link_libraries(${PROJECT_NAME}
   PUBLIC
   vm-bpf
   spdlog::spdlog
-  bpftime_frida_uprobe_attach_impl
-  bpftime_syscall_trace_attach_impl
+  bpftime_base_attach_impl
```

Comment:
Is it possible to remove this line? Seems base_attach_impl is a header only library.

Reply:
CMake use this way to use header only library